### DISCUSSION TITLE
ELRS Telemetry on Smartphone via bluetooth

### DISCUSSION THREAD

**Discussion by AQJay01** [ExpressLRS/ExpressLRS#1164] [Ideas]
This feature will enable the pilots to track their quads in realtime based on GPS telemetry received from the flight controller..
I think it would work as the BLE joystick feature works..
I dont know if anyone came up with this or already the devs are working..
Waiting to see thoughts of other fellow pilots:)

**Comment by Pairan** [comment]
... or forward the telemetry to an antenna tracker

**Comment by GusakIurii** [comment]
Greetings. Has anyone had success with this issue? Is it possible to track telemetry via the built-in bluetooth in ESP32 chips (ELRS transmitters)?

**Reply by lesharodin** [reply]
Just flash your tx with this PR https://github.com/ExpressLRS/ExpressLRS/pull/2305 and it works

**Reply by Rana0047** [reply]
Can you please help me with this ..exactly what i need to do

**Comment by druckgott** [comment]
Are there any news to get the expresslrs bluetooth working with Android Smartphone and for example with this app
https://github.com/RomanLut/android-taranis-smartport-telemetry

This Would be realy cool to parallel send data to the Smartphone and get it live. It works well for Radios with internal bluetooth But the Newer Radios did Not have anymore bluetooth like the Jumper t15

I see there is something on work ;-)
https://github.com/ExpressLRS/ExpressLRS/pull/2731
Maybe we can put focus on this.
